src/components/Form/Stories/SupportedControlsForm.js

```diff
@@ -89,6 +90,17 @@ export const SupportedControlsFormFields = [
       </div>
     )
   },
+  {
+    controlId: 'FieldLevelHelp',
+    label: 'FieldLevelHelp',
```

Comment:
same here: `Field Level Help`

Reply:
Addressed in f5628db87128706c1c309208f877c606cf4ae74f